packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options-modal.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { ChangeEvent, KeyboardEvent } from 'react';
+import { css } from '@emotion/css';
+import {
+  spacing,
+  uiColors,
+  Button,
+  Modal,
+  H3,
+  TextInput,
+  FormFooter,
+  Select,
+  Option,
+  OptionGroup,
+} from '@mongodb-js/compass-components';
+
+import FormFieldContainer from '../../form-field-container';
+import { editableUrlOptions, UrlOption } from '../../../utils/url-options';
+
+const modalContainerStyles = css({
+  padding: 0,
+  'button[aria-label="Close modal"]': {
```

Comment:
Hacky way of positioning the close button on modal. is there a better way?

Reply:
This should not be happening in theory, if we are hitting some issues with leafygreen like that we really need to try to get to the bottom of it (by asking in leafygreen-ui channel or raising in our private channels if help needed for debugging). We definitely can have these hacks and workaround if required if fix is not possible immediately, but we really need to have a hard reason for having these in the code. This doesn't seem to be happening to other modals we use in the app, so something about this particular use-case seems to be causing an issue 🤔

Reply:
I think this thread on slack might be similar to what we're seeing: https://mongodb.slack.com/archives/CFFA5BS79/p1639065827237500
If we bump our leafygreen versions it may go away.